rusoto/services/sts/src/custom/default_provider.rs

```diff
@@ -1,12 +1,12 @@
-use crate::WebIdentityProvider;
-use futures::{Future, Poll};
-use rusoto_core::credential::{
-    AutoRefreshingProvider, AutoRefreshingProviderFuture, AwsCredentialProviderChain,
-    AwsCredentials, AwsCredentialsProvider, ContainerProvider, CredentialsError,
-    EnvironmentProvider, InstanceMetadataProvider, ProfileProvider, ProvideAwsCredentials,
-};
-use std::rc::Rc;
-use std::time::Duration;
+//use crate::WebIdentityProvider;
+//use futures::{Future, Poll};
+//use rusoto_core::credential::{
+//    AutoRefreshingProvider, AutoRefreshingProviderFuture, AwsCredentialProviderChain,
+//    AwsCredentials, AwsCredentialsProvider, ContainerProvider, CredentialsError,
+//    EnvironmentProvider, InstanceMetadataProvider, ProfileProvider, ProvideAwsCredentials,
+//};
+//use std::rc::Rc;
+//use std::time::Duration;
```

Comment:
I don't really know the current state of STS stuff, but does STS need any additional fixing? And if not, should this just be removed?